chicagoland.pandemicresponsecommons.org/manifest.json

```diff
@@ -25,7 +25,7 @@
     "guppy": "quay.io/cdis/guppy:2020.07",
     "manifestservice": "quay.io/cdis/manifestservice:2020.07",
     "dashboard": "quay.io/cdis/gen3-statics:2020.07",
-    "auspice": "quay.io/cdis/gen3-auspice:v2.16.gen3"
+    "auspice": "quay.io/cdis/gen3-auspice:v2.16.gen3.1"
```

Comment:
https://github.com/uc-cdis/gen3-auspice/releases/tag/2.16.gen3.1
tag name is 2.16.gen3.1
we should have tag name starting with `v` -> `v2.16.gen3.1`